### ISSUE TITLE
Where to find easy_profiler_catkin?

### ISSUE THREAD

**Issue by ceinem** [jpl-x/x_events#1]
Hi, I'm trying to build this, can you point me to where I can find the catkin wrapper for easy_profiler?
Thank you!

**Comment by ceinem** [comment]
Let me rephrase, can you maybe make it public? cannot read from:
`git@fornat1.jpl.nasa.gov:x/x_external/x_easy_profiler_catkin.git`
Thanks!

**Comment by nubertj** [comment]
@jeff-delaune @florian-world 
Same issue here on my end. I tried to import it but it is not possible.
